### PR TITLE
[Term Entry] Java calendar getTime()

### LINKED ISSUE #3096: [Term Entry] Java Calendar getTime()
### Reason/inspiration (optional)

Please add the `getTime()` method as a new term entry for the [Calendar](https://www.codecademy.com/resources/docs/java/calendar)�class. Provide the entry in a markdown file and place under the file path `content/java/concepts/calendar/terms/getTime/getTime.md`

The entry should have the following:

�  An introductory paragraph about what the method is and does.
�  A syntax section with a pseudo code block with an explanation of it.
�  An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by ChiragAgg5k**
### Description

Added entry for getTime() method of Java Calendar class. Closes #3096 .

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/getTime/getTime.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.getTime()`** method of the `Calendar` class returns a `Date` object that represents time value of the `Calendar` class's object.
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getTime/getTime.md:15]:
```suggestion
The **`.getTime()`** method of the `Calendar` class returns a [`Date`](https://www.codecademy.com/resources/docs/java/date) object that represents time value of the `Calendar` class's object.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@ChiragAgg5k I did a review and only had one comment.

**Review by SSwiniarski — APPROVED** [review]
@ChiragAgg5k Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@ChiragAgg5k @SSwiniarski Just made a few small edits directly. Looks good to merge

**Pull request merged**